hub-config/tasks.json

```diff
@@ -1,25 +1,224 @@
+              "required": ["24A", "24B", "24C", "other", "recombinant"],
+              "optional": null
             }
+          },
+          "output_type": {
+            "mean": {
+              "output_type_id": {
+                "required": null,
+                "optional": [
+                  "NA"
+                ]
+              },
+              "value": {
+                "type": "double",
+                "minimum": 0,
+                "maximum": 1
+              }
+            },
+            "sample": {
+              "output_type_id_params": {
+                "is_required": false,
+                "type": "character",
+                "max_length": 15,
+                "min_samples_per_task": 100,
+                "max_samples_per_task": 100
+              },
+              "value": {
+                "type": "double",
+                "minimum": 0,
+                "maximum": 1
+              }
+            }
+          },
+          "target_metadata": [
+            {
+              "target_id": "variant prop",
+              "target_name": "Daily nowcasted variant proportions",
+              "target_units": "proportion",
+              "target_keys": null,
+              "target_type": "compositional",
+              "is_step_ahead": true,
+              "time_unit": "day"
+            }
+          ]
+        }
+      ],
+      "submissions_due": {
+        "relative_to": "nowcast_date",
+        "start": -2,
+        "end": 1
+      }
+    },
+    {
+      "round_id_from_variable": true,
+      "round_id": "nowcast_date",
+      "model_tasks": [
+        {
+          "task_ids": {
+            "nowcast_date": {
+              "required": [
+                "2024-09-18"
+              ],
+              "optional": null
+            },
+            "target_date": {
+              "required": null,
+              "optional": ["2024-08-18", "2024-08-19", "2024-08-20", "2024-08-21", "2024-08-22", "2024-08-23", "2024-08-24", "2024-08-25", "2024-08-26", "2024-08-27", "2024-08-28", "2024-08-29", "2024-08-30", "2024-08-31", "2024-09-01", "2024-09-02", "2024-09-03", "2024-09-04", "2024-09-05", "2024-09-06", "2024-09-07", "2024-09-08", "2024-09-09", "2024-09-10", "2024-09-11", "2024-09-12", "2024-09-13", "2024-09-14", "2024-09-15", "2024-09-16", "2024-09-17", "2024-09-18", "2024-09-19", "2024-09-20", "2024-09-21", "2024-09-22", "2024-09-23", "2024-09-24", "2024-09-25", "2024-09-26", "2024-09-27", "2024-09-28"]
+            },
+            "location": {
+              "required": null,
+              "optional": ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY", "PR"]
+            },
+            "variant": {
+              "required": ["24A", "24B", "24C", "other", "recombinant"],
```

Comment:
Noting that we appear to have lost our [alphabetical] + "other" ordering here. Not that it really matters.

Reply:
Yeah, I noticed that. We'll order the clade list in the `auxiliary-data` folder, but I don't think it's worth overriding whatever background tooling is re-ordering them when writing out the json.

Reply:
It's coming from https://github.com/reichlab/variant-nowcast-hub/pull/46/files#diff-3361d89a46e263279543b41cee56a8c99852d1f2052dd4bfe295d54842d57fdbR61

If you have python pre-sorting, you could remove `sort()` from this line

Reply:
That statement sorts the files in `auxiliary-data`, but shouldn't be affecting the order of the file contents (i.e., the clade list).

Reply:
The English major is stepping aside. Let's get this merged in.

Reply:
> That statement sorts the files in `auxiliary-data`, but shouldn't be affecting the order of the file contents (i.e., the clade list).

No, it is sorting the contents because it calls `get_clade_list()`

```
this_round_clade_list <- sort(get_clade_list(this_round_clade_file$clade_file))
```